core/src/main/resources/hudson/model/Messages_es.properties

```diff
@@ -120,11 +120,11 @@ Job.NOfMFailed={0} de las {1} \u00faltimas ejecuciones fallaron.
 Job.NoRecentBuildFailed=No hay ejecuciones recientes con fallos.
 Job.Pronoun=Proyecto
 Job.minutes=Min
-Job.NoRenameWhileBuilding=No es posible renombrar un proyecto mientras se est� ejecutando.
+Job.NoRenameWhileBuilding=No es posible renombrar un proyecto mientras se está ejecutando.
```

Comment:
GitHub refuses to show a diff here, but it seems you changed the encoding to UTF-8. It must be ISO-8859-1. Acc. to `jshell`,

```java
var p = new Properties()
p.load(new java.io.FileInputStream("/…/jenkinsci/jenkins/core/src/main/resources/hudson/model/Messages_es.properties"))
p.getProperty("Job.NoRenameWhileBuilding")
```

returns

```
No es posible renombrar un proyecto mientras se está ejecutando.
```

while the equivalent passing in the file as patched here produces mojibake

```
No es posible renombrar un proyecto mientras se estÃ¡ ejecutando.
```

Safest to throw away this modification and start over with an editor or tool that can safely handle ISO-8859-1 text.